docs-ref-services/keyvault-secrets-readme.md

```diff
@@ -518,33 +532,13 @@ setLogLevel("info");
 
 You can find more code samples through the following links:
 
-- [KeyVault Secrets Samples (JavaScript)](https://github.com/Azure/azure-sdk-for-js/tree/10c8acd23aa6716eed741fb796a23c7c7b084ca7/../../azure-sdk-for-js/sdk/keyvault/keyvault-secrets/samples/javascript)
-- [KeyVault Secrets Samples (TypeScript)](https://github.com/Azure/azure-sdk-for-js/tree/10c8acd23aa6716eed741fb796a23c7c7b084ca7/../../azure-sdk-for-js/sdk/keyvault/keyvault-secrets/samples/typescript)
-- [KeyVault Secrets Test Cases](https://github.com/Azure/azure-sdk-for-js/tree/10c8acd23aa6716eed741fb796a23c7c7b084ca7/../../azure-sdk-for-js/sdk/keyvault/keyvault-secrets/test)
+- [KeyVault Secrets Samples (JavaScript)](./samples/javascript)
+- [KeyVault Secrets Samples (TypeScript)](./samples/typescript)
+- [KeyVault Secrets Test Cases](./test/)
```

Comment:
In the above, the links changed to absolute links that linked to master. Here, these change to relative links. Why do we use both styles?

Reply:
This is my bad: https://github.com/Azure/azure-sdk-for-js/blame/master/sdk/keyvault/keyvault-secrets/README.md#L521 I should make these absolute links. I can do that in another PR to our repo! I wonder if that would make sense.

Reply:
I have updated these links!